sdk-core/src/main/java/care/data4life/sdk/network/model/NetworkModelContract.kt

```diff
@@ -20,51 +20,12 @@ import care.data4life.crypto.GCKey
 import care.data4life.sdk.data.DataResource
 import care.data4life.sdk.fhir.Fhir3Resource
 import care.data4life.sdk.fhir.Fhir4Resource
-import care.data4life.sdk.lang.CoreRuntimeException
 import care.data4life.sdk.lang.DataValidationException
 import care.data4life.sdk.tag.Annotations
 import care.data4life.sdk.tag.EncryptedTagsAndAnnotations
 import care.data4life.sdk.tag.Tags
 
 class NetworkModelContract {
```

Comment:
I notice that contracts are classes, why that?

Reply:
EncryptionService and Version are `internal`.